components/common/ScrollTop/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { useCallback } from 'react';
+
+import toScrollTop from '~/lib/toScrollTop';
+
+import * as S from './style';
+
+const ScrollTop = () => {
+  const onClick = useCallback(() => {
+    toScrollTop();
+  }, []);
```

Comment:
```suggestion
  const onClick = () => toScrollTop();
```
useCallback 으로 함수를 Wrapping 하는 것이 항상 최적화를 우선하는 것이 아닙니다.

Reply:
함수명을 리액트 이벤트명으로 짓는 것은 올바른 컨벤션이 아닌 것 같습니다.

suggestion: clickToScrollTop

Reply:
이 경우 리렌더링이 발생하지 않을 거라 생각되기 때문에 useCallback을 사용하지 않아도 되는건가요?

Reply:
올려주신 글 읽어보겠습니다

Reply:
쉽게 말해서, 
단순한 코드를 최적화하기 위해서 더 많은 연산이 수행된다는 겁니다.

Reply:
진짜 생각지도 못했던 부분이네요

Reply:
리뷰 주신부분 수정했습니다 알찬 리뷰 감사합니다